{{cookiecutter.project_slug}}/docs/LICENSE.md

```diff
@@ -0,0 +1 @@
+--8<-- "LICENSE.md"
```

Comment:
Not seen `--8<--` syntax before. Is this from some plugin?

Reply:
The `pymdownx.snippets` Markdown extension. I followed the suggestion in https://github.com/mkdocs/mkdocs/issues/2831